Binding revisions

holfordm opened the pull request.

holfordm committed:
Update customizations.xsl

new rules for new elements in binding descriptions

holfordm committed:
sample description + new customizations

-sample description (MS. Lyell 65)
-new rules in customizations.xsl for new elements

andrew-morrison commented:
You can define a prefix in the root element in the TEI record, same as in the XSLT, like this...
```XML
<TEI xml:id="manuscript_7799" xmlns="http://www.tei-c.org/ns/1.0" xmlns:msdesc="http://www.example.org/ns/msdesc">
```
...then this...
```XML
<spine xmlns="http://www.example.org/ns/msdesc">The binding has a slight round spine shape. No spine linings.</spine>
```
...could be replaced with this...
```XML
<msdesc:spine>The binding has a slight round spine shape. No spine linings.</msdesc:spine>
```
...and, because the default namespace would still be the TEI one, valid TEI tags could be used inside without re-declaring their namespace, like this...
```XML
<msdesc:spine>The binding has a slight round spine shape. <hi rend="bold">No spine linings.</hi></msdesc:spine>
```
It is all the same to an XML parser but more concise and clearer for humans.

In the XSLT, it would be better if the templates for each new element output a `<div>` instead of a `<p>`. That way, if they contain TEI `<p>` those would be transformed into HTML `<p>` without being nested inside another `<p>`, which is invalid (although web browsers will cope.)

holfordm committed:
Update MS_Lyell_65.xml

sample description: add attributes

holfordm committed:
Update customizations.xsl

holfordm committed:
Merge branch 'master' into binding-revisions

holfordm committed:
Update MS_Lyell_65.xml

adunning committed:
Merge branch 'master' into binding-revisions

Commit:
Update customizations.xsl

Commit:
sample files

holfordm committed:
Merge branch 'master' into binding-revisions

Commit:
Update MS_Bodl_392.xml

use verbose encoding

Commit:
Merge branch 'binding-revisions' of https://github.com/bodleian/medieval-mss into binding-revisions

Commit:
update schema references

holfordm committed:
Update customizations.xsl

remove styling (incompatible with Blacklight - modifications made to CSS instead)

Commit:
update descriptions

Commit:
Update MS_Lyell_65.xml

update and correct binding description

holfordm committed:
Merge branch 'master' into binding-revisions
